todo-cli.cabal

```diff
@@ -20,12 +20,18 @@ source-repository head
 library
   hs-source-dirs:       src
   build-depends:        base >= 4.9 && < 5
+                      , hashable >= 1.0.1.1 && < 1.3
```

Comment:
Btw, I just wonder, how do you determine version bounds for `.cabal` file? :slightly_smiling_face:

Reply:
Copied from dependencies list for [this](https://hackage.haskell.org/package/unordered-containers).